run-e5.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+
+import numpy as np
+from typing import Any
+from openai import AsyncOpenAI
+
+import mteb
+from mteb.model_meta import ModelMeta
+from mteb.models.text_formatting_utils import corpus_to_texts
+from mteb.models.instructions import task_to_instruction
+
+MODEL = "intfloat/e5-mistral-7b-instruct"
+TASKS = ["STS12"]
+
+class AsyncOpenAIWrapper:
+    def __init__(self, 
+                 model_name: str, 
+                 base_url: str = "http://localhost:8000/v1",
+                 num_concurrent: int = 100) -> None:        
+
+        self._client = AsyncOpenAI(base_url=base_url)
+        self._model_name = model_name
+        self._num_concurrent = num_concurrent
+
+    async def async_encode(self, sentences: list[str]) -> np.ndarray:
+        sublists = [
+            sentences[i : i + self._num_concurrent]
+            for i in range(0, len(sentences), self._num_concurrent)
+        ]
+
+        all_embeddings = []
+
+        for sublist in sublists:
+            outputs = [
+                self._client.embeddings.create(
+                    input=sentence,
```

Comment:
Why not sending `sublist` here instead of doing a loop? Is it because you want to verify the correctness of passing one vs. multiple strings?